pkg/infoschema/infoschema_v2.go

```diff
@@ -871,6 +871,25 @@ func applyDropSchema(b *Builder, diff *model.SchemaDiff) []int64 {
 }
 
 func applyRecoverSchema(b *Builder, m *meta.Meta, diff *model.SchemaDiff) ([]int64, error) {
+	if snapTS := diff.RecoverSnapshotTS; snapTS > 0 {
```

Comment:
i prefer reload from now, there is no difference.

Reply:
Just in case the database is relatively small compared with full load.

Reply:
why full load? i mean load using current ts, not this saved one

Reply:
https://github.com/pingcap/tidb/pull/54665/commits/8f5520c93e14a845ac39c84aa35b654f90d774a7